packages/svelte-ux/src/routes/docs/components/Table/+page.svelte

```diff
@@ -201,6 +203,51 @@
   />
 </Preview>
 
+<h2>Order + Pagination</h2>
+
+<Preview>
+  <Paginate items={sorted_data} perPage={5} let:pageItems let:pagination>
+    <Table
+      data={pageItems}
+      columns={[
+        {
+          name: 'name',
+          align: 'left',
+        },
+        {
+          name: 'calories',
+          align: 'right',
+          format: 'integer',
+        },
+        {
+          name: 'fat',
+          align: 'right',
+          format: 'integer',
+        },
+        {
+          name: 'carbs',
+          align: 'right',
+          format: 'integer',
+        },
+        {
+          name: 'protein',
+          align: 'right',
+          format: 'integer',
+        },
+      ]}
+      orderBy={$order.by}
+      orderDirection={$order.direction}
+      on:headerClick={(e) => order.onHeaderClick(e.detail.column)}
```

Comment:
When changing the sort order (via `onHeaderClick`), it usually makes sense to reset back to page one (if you were on say page 2 or 3).

Something like this should work:
```svelte
on:headerClick={(e) => {
  pagination.setPage(1);
  order.onHeaderClick(e.detail.column)
}}
```